packages/domain/lib/usecase/get_trending_movies_use_case.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:domain/model/movie_trending.dart';
+import 'package:domain/repository/base_repository.dart';
+import 'package:domain/usecase/use_case.dart';
```

Comment:
in the whole project, you have the "usecase" import highlighted, change the name - "use_case"